pyccl/halos/profiles.py

```diff
@@ -0,0 +1,824 @@
+          for the upper end of the range (e.g. a value of
+          10 implies extending the range by one decade on
+          the right). Default value: 10.
+        * `n_per_decade`: number of samples of the profile
+          taken per decade when computing Hankel transforms.
+        * `padding_lo_extra`: when computing the projected
+          2D profile or the 2D cumulative density,
+          sometimes two Hankel transforms are needed (from
+          3D real-space to Fourier, then from Fourier to
+          2D real-space). This parameter controls the k range
+          of the intermediate transform. The logic here is to
+          avoid the range twice by `padding_lo_fftlog` (which
+          can be overkill and slow down the calculation).
+          Default value: 0.1.
+        * `padding_hi_extra`: same as `padding_lo_extra` for
+          the upper end of the range. Default value: 10.
+        * `large_padding_2D`: if set to `True`, the
+          intermediate Hankel transform in the calculation of
+          the 2D projected profile and cumulative mass
+          density will use `padding_lo_fftlog` and
+          `padding_hi_fftlog` instead of `padding_lo_extra`
+          and `padding_hi_extra` to extend the range of the
+          intermediate Hankel transform.
+        * `extrapol`: type of extrapolation used in the uncommon
+          scenario that FFTLog returns a profile on a range that
+          does not cover the intended output range. Pass
+          `linx_liny` if you want to extrapolate linearly in the
+          profile and `linx_logy` if you want to extrapolate
+          linearly in its logarithm. Default value: `linx_liny`.
+        * `plaw_fourier`: FFTLog is able to perform more
+          accurate Hankel transforms by prewhitening its arguments
+          (essentially making them flatter over the range of
+          integration to avoid aliasing). This parameter
+          corresponds to a guess of what the tilt of the profile
+          is (i.e. profile(r) = r^tilt), which FFTLog uses to
+          prewhiten it. This parameter is used when computing the
+          real <-> Fourier transforms. The methods
+          `_get_plaw_fourier` allows finer control over this
+          parameter. The default value allows for a slightly faster
+          (but potentially less accurate) FFTLog transform. Some
+          level of experimentation with this parameter is
+          recommended when implementing a new profile.
+          Default value: -1.5.
+        * `plaw_projected`: same as `plaw_fourier` for the
+          calculation of the 2D projected and cumulative density
+          profiles. Finer control can be achieved with the
+          `_get_plaw_projected`. The default value allows for a
+          slightly faster (but potentially less accurate) FFTLog
+          transform.  Some level of experimentation with this
+          parameter is recommended when implementing a new profile.
+          Default value: -1.
```

Comment:
OK, I'm not familiar with this part of sphinx. Do you know what the standard is here? (I'm looking into it, but just in case you know off the top of your head)

Reply:
Just make these look like any other doc string in the repo.

Reply:
e.g, no `*` etc

Reply:
Done!